README.md

```diff
@@ -53,9 +53,71 @@ public class JaxrsApplication extends Application {
 Just define them as Spring beans, and they will be automatically registered.
 Notice that JAX-RS resources can be singleton or request scoped, while JAX-RS providers must be singletons.
 
+
 ### Further information
 See [How to use RESTEasy Spring Boot Starter](mds/USAGE.md).
 
+
```

Comment:
Please, move the whole testing end user documentation under md file **mds/USAGE.md**